simple/dgram.c

```diff
@@ -76,39 +76,23 @@ static int init_fabric(void)
 
 static int send_recv()
 {
-	struct fi_cq_entry comp;
 	int ret;
 
 	if (opts.dst_addr) {
 		fprintf(stdout, "Sending message...\n");
 		sprintf(buf, "Hello from Client!");
```

Comment:
Right, using the data verification method would use a generated message on send/recv side rather than the hardcoded "hello..." message. If that change was made then I guess it wouldn't make sense for `fi_dgram` to print what it received anymore. It could just send/recv a message and verify that the message is correct.